src/containers/MemoryGame/index.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Navbar from "../../components/Navbar";
+import "./index.css";
+
+const MemoryGame = () => {
+  return (
+    <div className="memoryGameContainer">
+      <Navbar />
```

Comment:
i don't think you should be having to declare the Navbar on every page...seems wrong. it should be declared in a top-level component, and using routing the thing you want to show should show up.

Reply:
fixed